tests/test_filesystem.py

```diff
@@ -96,3 +96,52 @@ def test_filesystem_zip__with_source_prefix():
     for source_file in source_files_in_zip:
         with fs.open(source_file) as f:
             assert hasattr(f, 'read')
+
+
+def test_filesystem_git():
+    import pycobertura.filesystem as fsm
+    from pycobertura.filesystem import GitFileSystem
+
+    branch, folder, filename = "master", "tests/dummy", "test-file"
+
+    with patch.object(fsm, "subprocess") as subprocess_mock:
+        subprocess_mock.check_output = MagicMock(return_value=b"<file-content>")
+
+        fs = GitFileSystem(folder, branch)
+        git_filename = fs.real_filename(filename)
+
+        with fs.open(filename) as f:
+            assert hasattr(f, 'read')
+
+        assert git_filename.startswith(branch)
```

Comment:
Is it possible to assert for the whole filename just to be sure it contains everything necessary? For example:

```suggestion
        expected_git_filename = "master:tests/dummy/test-file"
        assert git_filename == expected_git_filename 
```